gem_tasks/cucumber.rake

```diff
@@ -3,25 +3,8 @@
 require 'cucumber/rake/task'
 require 'cucumber/platform'
 
-class Cucumber::Rake::Task
```

Comment:
Shouldn't this code removal be on another PR? I don't see how removing this code helps to fix a backwards compability problem.